src/app/(main)/hosts/[hostListingId]/tenant-schemas.ts

```diff
@@ -5,6 +5,6 @@ export const tenantRequestFormSchema = z.object({
   itemsDescription: z.string().trim().min(1),
   sqft: z.coerce.number().min(1, "The square footage cannot be zero"),
   startTime: z.coerce.date(),
-  duration: z.coerce.number().min(1, "The duration cannot be 0"),
+  endTime: z.coerce.date(),
```

Comment:
We can use [`superRefine`](https://github.com/colinhacks/zod) for validation across more than 1 field (for example, confirming password by having user type it twice, or ensuring that the end time is after the start time in this case)

Reply:
I had an idea that might be possible, but I wasn't sure